setup.py

```diff
@@ -58,6 +58,19 @@
 datasets_extra_require = [
     "pyarrow >= 3.0.0, < 8.0dev",
 ]
+
+vizier_extra_require = [
+    "attrs==21.4.0",
+    "absl-py>=0.7",
+    "numpy>=1.19.0",
+    "protobuf>=3.6,<4.0",
+    "keras-tuner>=1.0,<2.0",
+    "portpicker==1.3.1",
+    "googleapis-common-protos==1.56.0",
+    "google-api-python-client==1.12.8",
+    "google-vizier==0.0.3a",
```

Comment:
Are there any issues with supporting higher versions of vizier? 0.0.3a seems to be quite behind 0.0.15